### PR TITLE
Math-magician-app: Calculator component

### PR DESCRIPTION
**Pull request by BertrandConxy**
# Math-magician-app: Calculator component
In this branch, I have done the following:
- Deleted unnecessary files
- Created UI for the calculator using class component
- Styled the calculator to look like the design given.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (Mihndim2020) [inline src/App.js]: - [ ] Please as it is a requirement for this project, all your components should be class based, please convert you `App.js` component to a class-based component. See screenshot of the requirements below for clarifications.

ClassBased

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @BertrandConxy, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages,
:heavy_check_mark: Descriptive Readme file,
:heavy_check_mark: Your app is running free of `Linter` errors.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (BertrandConxy) [reply src/App.js]: Resolved

**Review by elyor-doniyorov — APPROVED** [review]
Hi @BertrandConxy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BertrandConxy** [comment]
Thank you @elyor-doniyorov. I appreciate your review.

**Pull request merged**